packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -115,6 +134,48 @@ class DatePickerInput extends React.Component {
     }
   }
 
+  static getDerivedStateFromProps(props, state) {
+    const format = DateUtil.getDateFormat(props.intl.formatMessage({ id: 'Terra.datePicker.dateFormatOrder' }));
+    const tempDate = DateUtil.convertToISO8601(props.value, format);
+
+    if (tempDate && props.value !== state.prevDate) {
+      return {
+        day: tempDate.substr(8, 2),
+        month: tempDate.substr(5, 2),
+        year: tempDate.substr(0, 4),
+        prevDate: props.value,
+      };
+    }
+    return null;
+  }
+
+  moveFocusOnChange(inputValue, type, dateVariant) {
```

Comment:
Please add documentation for the arguments here so that it's easier for the readers to understand in the future.

Reply:
Added! [89c3693](https://github.com/cerner/terra-framework/pull/1004/commits/89c3693f31162adfc53f90f681cdfda21588b6b0)